core/tests/protractor_utils/ExplorationEditorSettingsTab.js

```diff
@@ -169,7 +166,11 @@ var ExplorationEditorSettingsTab = function() {
     await waitFor.presenceOf(
       explorationCategoryInput,
       'Exploration category input takes too long to be visible.');
-    expect(await explorationCategoryInput.$('option:checked').getText()).
+    var checkedElement = explorationCategoryInput.$('option:checked');
+    await waitFor.visibilityOf(
+      checkedElement,
+      'Checked Element takes too long to be visible.');
+    expect(await checkedElement.getText()).
```

Comment:
We now have an `action.getText` function you can use instead

Reply:
Done